.github/install-codecov.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
The file name is `install-codecov.sh`, but you are not installing codecov here. You are installing `kcov`. Hence, `install-kcov.sh` would be more suitable.

Reply:
Removed the scripts.